Source/DOM classes/Unported or Partial DOM/SVGImageElement.m

```diff
@@ -105,7 +106,7 @@ - (CALayer *) newLayer
 	 */
 	AppleNativeImageRef image = [AppleNativeImage imageWithData:imageData];
 	
-    if( image == nil ) // NSData doesn't contain an imageformat Apple supports; might be an SVG instead
+    if( image == nil && imageURL ) // NSData doesn't contain an imageformat Apple supports; might be an SVG instead
```

Comment:
OK, but that bug needs to be fixed. Breaking this for all versions of iOS isn't workable!

Do we have a bug report (even better: a workaround/fix?) for the ios8 crash? Is it simply this line that's causing it:

NSInputStream* stream = [NSInputStream inputStreamWithURL:u];